lib/jbuilder/jbuilder_template.rb

```diff
@@ -111,9 +111,8 @@ def _render_partial_with_options(options)
     if as && options.key?(:collection)
       as = as.to_sym
       collection = options.delete(:collection)
-      locals = options.delete(:locals)
       array! collection do |member|
-        member_locals = locals.clone
+        member_locals = options.clone
```

Comment:
Does it mean using locals won't work now?
```
json.comments @post.comments, partial: 'comments/comment', locals: { show: false }, as: :comment
```

Reply:
I wrote a spec that I'm pretty sure confirms your suspicion that locals don't work now but haven't pushed it. I've been pretty swamped the last couple weeks but should be much freer next week to take a more in depth look. Thanks for taking a look @yusufali2205!